utils/typography.js

```diff
@@ -28,7 +28,7 @@ const typography = new Typography({
     'Verdana',
     'sans-serif'
   ],
-  scale: 1.5,
+  scaleRatio: 1.5,
```

Comment:
Cool! Always fun to see things being used on in the wild :-) Need to do another push to incorporate some more typography things I've learned into the library and get it to a 1.0.